va/va.h

```diff
@@ -1111,6 +1115,21 @@ typedef union _VAConfigAttribValEncRateControlExt {
     uint32_t value;
 } VAConfigAttribValEncRateControlExt;
 
+/** \brief Attribute value for VAConfigAttribMultipleFrame*/
+typedef union _VAConfigAttribValMultipleFrame {
+    struct {
+        /** \brief max num of concurrent frames from different stream */
+        max_num_concurrent_frames      : 8;
+        /** \brief indicate whether MF support mixed codecs */
+        mixed_codecs                   : 1;
```

Comment:
I believe it would be great if the new interface can say which combination of codec is supported.

Reply:
see, but 32 bit is hard to describe it. I dont want to declare some Macro such as CODEC_H264 CODEC_HEVC  which can be OR'd together, because there are vaProfileXXXX already.

Reply:
Then, IMO, it's useless because application won't be able to build a proper logic using  bool 'mixed_codecs'. In such case it's better to simply remove it.

Reply:
thanks, agree, will remove it

Reply:
hi Carl, 
last time we discussed to remove mixed_codecs for now, right? could you remove it?